backend/typescript/rest/authRoutes.ts

```diff
@@ -43,9 +43,19 @@ authRouter.post("/login", loginRequestValidator, async (req, res) => {
   }
 });
 
+/* returns a firebase user given a user id */
+authRouter.get("/:uid", async (req, res) => {
```

Comment:
Did you miss adding this end-point to the swagger.yml?

Reply:
@Tahmeed27 do we need to protect this endpoint? We need it so that unauthorized users and even unregistered users can get the firebaseUser but as of now, anyone can call this endpoint to get our user right?

Reply:
Does it expose anything sensitive? If it's just Firebase User, then it should be fine right?